src/sagemaker/serve/builder/transformers_builder.py

```diff
@@ -161,7 +164,7 @@ def _get_hf_metadata_create_model(self) -> Type[Model]:
                 vpc_config=self.vpc_config,
             )
 
-        if not self.image_uri and self.mode == Mode.LOCAL_CONTAINER:
+        if self.mode == Mode.LOCAL_CONTAINER or self.mode == Mode.IN_PROCESS:
```

Comment:
nit:
```
LOCAL_MODES = [Mode.LOCAL_CONTAINER, Mode.IN_PROCESS]
if self.mode in LOCAL_MODES:
      ...
```

Reply:
Good suggestion, I've made the edit, thank you